README.md

```diff
@@ -196,19 +196,31 @@ There are a few CLI utilities included:
     - Nextcloud data
 
 #### What about ZFS?
+
 - Zpools may be bind mounted to accomplish data access by the snap 
     - For example, create a new directory (e.g. `mkdir -p /mnt/zpool_nc`)
     - Depending on where your zpool exists (e.g. `/pool/tank01`), you can create a
       bind mount to allow the snap to acces/manage your data
-        - For example: `mount --bind /mnt/zpool_nc /pool/tank01`
+        - For example: `mount bind -o /mnt/zpool_nc /pool/tank01`
```

Comment:
This is incorrect. It should be 

`mount --bind /pool/tank0 /var/snap/nextcloud/common/nextcloud/data`

also see: https://github.com/nextcloud-snap/nextcloud-snap/issues/2322